synth.py

```diff
@@ -18,25 +18,60 @@
 import synthtool.gcp as gcp
 import logging
 import subprocess
+import json
+import os
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
-version = 'v1'
-library = gapic.node_library(
-        'kms', version,
-        config_path='artman_cloudkms.yaml')
-# skip index, package.json, and README.md
-s.copy(
-    library,
-    excludes=['package.json', 'README.md', 'src/index.js'],
-)
+gapic = gcp.GAPICMicrogenerator()
+versions = ['v1']
+for version in versions:
+    library = gapic.typescript_library(
+            'kms', version,
+            generator_args={
+                "grpc-service-config": f"google/cloud/kms/{version}/cloudkms_grpc_service_config.json",
+                "package-name": f"@google-cloud/kms"
+                },
+            proto_path=f'/google/cloud/kms/{version}',
+            extra_proto_files=['google/cloud/common_resources.proto', 'google/iam/v1']
+            )
+    # skip index, package.json, and README.md
+    s.copy(
+        library,
+        excludes=['package.json', 'README.md', 'src/index.ts', 'src/helper.ts', 'tslint.json'],
```

Comment:
Thinking we don't want `helper.ts` or `tslint.json` here.

Reply:
removed them, thank you